src/main/resources/swagger/api-docs.yaml

```diff
@@ -5948,6 +6017,52 @@ definitions:
       methodConfiguration:
         $ref: '#/definitions/MethodConfiguration'
 
+  WorkflowCost:
+    description: The cost of a workflow, if available
+    properties:
+      cost:
+        type: number
+        description: Workflow cost
+      workflowId:
+        type: string
+        description: The id of this workflow
+
+  Workflow:
+    description: ''
+    properties:
+      workflowId:
+        type: string
+        description: Workflow unique identifier
+      status:
+        $ref: '#/definitions/WorkflowStatus'
+        description: Workflow status
+      statusLastChangedDate:
+        type: string
+        format: date-time
+        description: Status last-changed date
+      workflowEntity:
+        $ref: '#/definitions/AttributeEntityReference'
+        description: Optional if this workflow did not run on an entity (i.e. its method configuration has no root entity)
+      inputResolutions:
+        type: array
+        items:
+          $ref: '#/definitions/SubmissionValidationValue'
+        description: The evaluation of each input
+      messages:
+        type: array
+        description: List of messages about this workflow
+        items:
+          type: string
+      cost:
+        type: number
+        format: float
+        description: The run cost of this workflow
+
+  WorkflowStatus:
+    properties:
+      done:
```

Comment:
remove this.  it's an enum like SubmissionStatus so `name` is what's relevant.  Reuse SubmissionStatus' format.